golos.referral/src/golos.referral.cpp

```diff
@@ -6,6 +6,25 @@ namespace golos {
 
 using namespace eosio;
 
+extern "C" {
```

Comment:
We have `DISPATCH_WITH_TRANSFER ` macro for this:
https://github.com/GolosChain/golos-smart/blob/d519804e0b97538eb727ce69d3be95291a0a13bf/golos.ctrl/src/golos.ctrl.cpp#L347